src/components/panes/AddViewColumnPane.jsx

```diff
@@ -238,16 +238,18 @@ class PersonFieldColumnTemplate extends React.Component {
 class PersonNotesColumnTemplate extends React.Component {
 
     componentDidMount() {
-        const column = {
-            title: this.props.intl.formatMessage({
-                id: 'panes.addViewColumn.templates.person_notes.title'
-            }),
-            config: {
-                limit: 1,
-            }
-        };
+        if(this.props.selected) {
```

Comment:
This will never be true, because the component only mount once, and it's not `selected` when it first mounts. It *might* work putting this in a `componentWillRecieveProps()` instead.